package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@osjs/client",
-  "version": "3.7.3",
+  "version": "3.7.4",
```

Comment:
Please revert this. Version bumps are done exclusively in a separate commit along with changelogs and git tags :)